libnmstate/nm/ipv4.py

```diff
@@ -51,6 +55,8 @@ def create_setting(config, base_con_profile):
             setting_ipv4.props.method = (
                 nmclient.NM.SETTING_IP4_CONFIG_METHOD_MANUAL)
             _add_addresses(setting_ipv4, config['address'])
+
+        nm_route.add_routes(setting_ipv4, config.get('routes', []))
```

Comment:
This should probably be a constant.

Reply:
Changed to `_routes` as you suggested for meta data name.